microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -191,23 +200,20 @@ def verify_gpu_cuda_with_pytorch(self, node: Node) -> None:
 
 
 def _check_driver_installed(node: Node) -> None:
+
+    ##os.system("./microsoft/runbook/azure/install_dependencies_alma.sh alma8.6")
+
     gpu = node.features[Gpu]
 
     if not gpu.is_supported():
         raise SkippedException(f"GPU is not supported with distro {node.os.name}")
     if ComputeSDK.AMD in gpu.get_supported_driver():
         raise SkippedException("AMD vm sizes is not supported")
-
-    if isinstance(node.os, (Suse, AlmaLinux, Oracle)):
-        raise SkippedException(
-            f"{node.os.name} doesn't support GPU driver installation."
-        )
-
     try:
         _ = node.tools[NvidiaSmi]
     except Exception as identifier:
         raise LisaException(
-            f"Cannot find nvidia-smi, make sure the driver installed correctly. "
+            f"Cannot find nvidia-smi {node.os.name}, make sure the driver installed correctly. "
```

Comment:
os name info will be available in the logs. We avoid redundancy in code and avoid logging values which can be inferred from the logs

Reply:
I added it for debugging purposes. Forgot to remove. Just done.